Sources/Jobs/Scheduler/DateExtensions/DateExtensionComparison.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+extension Date {
+    /// Returns the amount of years from another date
+    func years(from date: Date) -> Int? {
+        return Calendar.current.dateComponents([.year], from: date, to: self).year ?? nil
```

Comment:
No need for the nil coalescing operator here: `return Calendar.current.dateComponents([.year], from: date, to: self).year`